app/assets/javascripts/staff/program/grid.js

```diff
@@ -5,7 +5,7 @@
   if (typeof(window.Schedule.Grid) !== 'undefined') {
     return window.Schedule.Grid;
   }
-
+  
```

Comment:
Looks like just a straggling space here to remove...I think. Not sure if that's only on the github side PR view.